var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -289,6 +289,8 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     # https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10113
     patch("adios2-remove-deprecated-functions.patch", when="@5.10: ^adios2@2.9:")
 
+    patch("exodusII-netcdf4.9.0.patch", when="@:5.10.1")
```

Comment:
```suggestion
    patch("exodusII-netcdf4.9.0.patch", when="@:5.10.2")
```

This patch did not seem to make it to the 5.10.2 release.

Reply:
True, I didn't see that version in spack so I did added it. I'll add it.